stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -50,6 +55,12 @@ def _compute_created_inventory_count(self):
         readonly=True, states={'wait': [('readonly', False)]},
         track_visibility="onchange",
     )
+    company_id = fields.Many2one(
+        comodel_name='res.company', string='Company',
+        required=True,
+        default=_default_company,
```

Comment:
Is this enough to be upgraded, no need for migration scripts?

Reply:
Yes, I think it's enough with an upgrade to pick the default company